fvm/src/kernel/mod.rs

```diff
@@ -345,3 +359,12 @@ pub trait DebugOps {
     /// Returns error on malformed name, returns Ok and logs the error on system/os errors.
     fn store_artifact(&self, name: &str, data: &[u8]) -> Result<()>;
 }
+
+/// Environment ops
+pub trait EnvOps {
```

Comment:
For better naming consistency.

```suggestion
pub trait ChainOps {
```

Reply:
ok.

Reply:
done.